js-miniapp-bridge/src/types/purchaseProduct.ts

```diff
@@ -0,0 +1,19 @@
+/** Points type. */
+export interface Product {
+  id: string;
+  title: string;
+  description: string;
+  price: ProductPrice;
+}
+
+export interface ProductPrice {
+  amount: number;
+  currencyCode: string;
+  price: string;
```

Comment:
In Android, there is an object e.g. [SkuDetails](https://developer.android.com/reference/com/android/billingclient/api/SkuDetails) returns price as micro, if iOS has a similar thing, we can add a new field for that in `ProductPrice`.